.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build_deploy:
+
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-latest
    environment: publishing
```

Reply:
Anywhere you all keep standards for this stuff? It feels like it's changing under me which is fine I guess if you'll correct me :D